### PR TITLE
Update to version 1.1.1

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Add myself to the maintainers, as the PR has been open for 3 months, and the current versions are piling up.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@sugatoray @conda-forge/core could you please review and merge - 
I've added myself to the recipe as a maintainers, as there 6 open PRs and current conda pr version is already 9 release behind from pip / source. I will try to help bring this feedstock back to active and current maintenance.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,84 +1,74 @@
...
+    ## Converting the recipe to noarch: only build for python>=3.7
+    - backports.cached-property
+    - exceptiongroup
+    - importlib-metadata
 
 test:
   imports:
     - bentoml
   commands:
-    - pip check
```
Comment (xhochy) [inline recipe/meta.yaml]: You shouldn't remove this.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (rxm7706) [reply recipe/meta.yaml]: Oh I didnt notice that.. thanks for catching that @xhochy 
let me revert, and retest - Not sure why or how that happened, this PR was done 3 months ago.

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@xhochy - Ready for review.

**Comment by sugatoray** [comment]
@rxm7706 Thank you for the PR. Looks good to me. 🚀

**Review by xhochy — APPROVED** [review]

**Comment by rxm7706** [comment]
> @rxm7706 Thank you for the PR. Looks good to me. 🚀

Awesome - could you go ahead merge please !

**Pull request merged**